tools/topology/topology2/cavs/cavs-rt5682.conf

```diff
@@ -0,0 +1,274 @@
+<searchdir:cavs>
+<searchdir:include>
+<searchdir:include/common>
+<searchdir:include/components>
+<searchdir:include/dais>
+<searchdir:include/pipelines/cavs>
+<searchdir:cavs/platform/intel>
+
+<vendor-token.conf>
+<manifest.conf>
+<pdm_config.conf>
+<tokens.conf>
+<virtual.conf>
+<passthrough-capture.conf>
+<passthrough-playback.conf>
+<passthrough-be.conf>
+<host-copier-gain-mixin-playback.conf>
+<mixout-gain-dai-copier-playback.conf>
+<data.conf>
+<pcm.conf>
+<pcm_caps.conf>
+<fe_dai.conf>
+<ssp.conf>
+<dmic.conf>
+<hda.conf>
+<hw_config.conf>
+<manifest.conf>
+<route.conf>
+<cavs/common_definitions.conf>
+<copier.conf>
+<pipeline.conf>
+<dai.conf>
+<host.conf>
+<dmic-default.conf>
+
+<hdmi-default.conf>
+
+Define {
+	MCLK 				24576000
+	# override DMIC default definitions
+	NUM_DMICS			0
+	DMIC0_ID			1
+	DMIC1_ID			2
+	DMIC0_HOST_PIPELINE_ID		13
+	DMIC0_DAI_PIPELINE_ID		14
+	DMIC0_HOST_PIPELINE_SINK	'copier.host.13.1'
+	DMIC0_DAI_PIPELINE_SRC		'copier.DMIC.14.1'
+	DMIC0_NAME			'dmic01'
+	DMIC1_NAME			'dmic16k'
+	DMIC0_PCM_CAPS			'Passthrough Capture 13'
+	DMIC0_PIPELINE_STREAM_NAME	'copier.DMIC.14.1'
+	PLATFORM 			"none"
+	# override BE link ids
+	NUM_HDMIS			0
+	HDMI1_ID			3
+	HDMI2_ID			4
+	HDMI3_ID			5
+	HDMI4_ID			6
+}
+
+# override defaults with platform-specific config
+IncludeByKey.PLATFORM {
+	"mtl"	"platform/intel/mtl.conf"
+}
+
+# include DMIC config if needed.
+IncludeByKey.NUM_DMICS {
+	"[1-4]"	"platform/intel/dmic-generic.conf"
+}
+
+# include HDMI config if needed.
+IncludeByKey.NUM_HDMIS {
+	"[3-4]"	"platform/intel/hdmi-generic.conf"
+}
+
+#
+# List of all DAIs
+#
+#SSP Index: 0, Direction: duplex
+Object.Dai {
+	SSP."0" {
+		id 		0
+		direction	"duplex"
+		name		SSP0-Codec
+		default_hw_conf_id	0
+		sample_bits		32
+		io_clk		$MCLK
+
+		Object.Base.hw_config."SSP0" {
+			id	0
+			mclk_freq	$MCLK
+			bclk_freq	3072000
+			tdm_slot_width	32
+		}
+	}
+	SSP."1" {
+		id 		7
+		dai_index	1
+		direction	"playback"
+		name		SSP1-Codec
+		default_hw_conf_id	0
+		sample_bits		32
+		io_clk		$MCLK
+
+		Object.Base.hw_config."SSP1" {
+			id	0
+			mclk_freq	$MCLK
+			bclk_freq	3072000
+			tdm_slot_width	32
+		}
+	}
+}
+
+#
+# Pipeline definitions
+#
+# PCM0 ---> gain ----> Mixin ----> Mixout ----> gain ----> SSP0
+# PCM1 ---> gain ----> Mixin ----> Mixout ----> gain ----> SSP1
+#
+# SSP0 ----> PCM0
+# SSP1 ----> PCM1
```

Comment:
this inconsistent with the multiple-stream requirement on capture. Is the plan to add additional copiers and PCMs later for capture?

Reply:
Yes, plan to add DeepBuffer and multi-stream later later incrementally, also re-use the same file to support Rex0 variant, thank you for review.